### PR TITLE
Milestone 3: set up associations

### PR DESCRIPTION
**Pull request by abel-tefera**
### Summary

- Created a class `Classroom` with has-many/belongs-to relationship with Student
- Created classes `Book` and `Rental` with `Book` having many-to-many relationship with `Person`
- Used `Rental` class as an intermediate class for many-to-many relationship between `Person` and `Book`

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def rentals(person, date)
+    Rental.new(date, self, person)
+  end
+
+  attr_accessor :title, :author
```
Comment (elisha2kyakpo1) [inline book.rb:12]: - [ ] A rental should be written/read in the book class. Kindly add rentals to the attr_accessor.

File: person.rb [hunk 2]
```diff
@@ -7,6 +7,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
   attr_accessor :name, :age
```
Comment (elisha2kyakpo1) [inline person.rb:12]: - [ ]  A rental should be written/read in the book class. Kindly add rentals to the attr_accessor.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @abel-tefera, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description are good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.
✔️ Associations are nicely done👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: book.rb (hunk 1 above)
Comment (abel-tefera) [reply book.rb:12]: @elisha2kyakpo1 I've already created a method to set the rentals in the person and book classes

**Review by DeliceLydia — APPROVED** [review]
## Hi @abel-tefera, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

## Highlights
- Addressed changes requested


### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**